cli/tests/upgrade-tests/main_test.go

```diff
@@ -0,0 +1,73 @@
+package upgrade_tests
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"testing"
+	"time"
+
+	test_utils "github.com/timescale/tobs/cli/tests/test-utils"
+)
+
+var PATH_TO_TOBS = "./../../bin/tobs"
+var PATH_TO_CHART = "./../../../chart/"
+var NAMESPACE = "ns"
+var RELEASE_NAME = "gg"
+
+func TestMain(m *testing.M) {
+	// Signal handling
+	sigchan := make(chan os.Signal, 1)
+	done := make(chan bool, 1)
+	signal.Notify(sigchan, os.Interrupt)
+	go func() {
+		<-sigchan
+		done <- true
+		os.Exit(1)
+	}()
+
+	installTobsRecentRelease()
+
+	upgradeTobsLatest()
```

Comment:
Not sure why these are in TestMain and not in a test run by m.Run?

Reply:
`m.Run` will run all the test files under `upgrade-tests` in the future we might add more tests to this. So the main `install` & `upgrade` flow are global and the rest of the tests will run on top of it.

Reply:
Hmm not quite sure that will work as, for example, TestUpgrade leaves the system after it's run in a very different state than upgradeTobsLatest(). Therefore any test that runs after TestUpgrade will not be able to rely on the state. I suspect you'll want to do this kind of setup in each test individually. But, we can leave it like this for now and see how it goes.

Reply:
As the current tests. Do not need this install & upgrade to be outside. So I will move them as you suggested. Maybe in the future, I will pull them out. If I find more upgrade tescases.